src/metabase/embed/settings.clj

```diff
@@ -0,0 +1,32 @@
+(ns metabase.embed.settings
+  "Settings related to embedding Metabase in other applications."
+  (:require
+   [metabase.analytics.snowplow :as snowplow]
+   [metabase.api.common :as api]
+   [metabase.models.setting :as setting :refer [defsetting]]
+   [metabase.public-settings :as public-settings]
+   [metabase.util.i18n :as i18n :refer [deferred-tru]]
+   [toucan2.core :as t2]))
+
+(defsetting embedding-app-origin
+  (deferred-tru "Allow this origin to embed the full {0} application"
+                (public-settings/application-name-for-setting-descriptions))
+  :feature    :embedding
+  :visibility :public
+  :audit      :getter)
+
+(defsetting enable-embedding
+  (deferred-tru "Allow admins to securely embed questions and dashboards within other applications?")
+  :type       :boolean
+  :default    false
+  :visibility :authenticated
+  :audit      :getter
+  :setter     (fn [new-value]
+                (when (not= new-value (setting/get-value-of-type :boolean :enable-embedding))
+                  (setting/set-value-of-type! :boolean :enable-embedding new-value)
+                  (let [snowplow-payload {:embedding-app-origin-set   (boolean (embedding-app-origin))
+                                          :number-embedded-questions  (t2/count :model/Card :enable_embedding true)
```

Comment:
Out of curiosity, will this number be meaningful? Are people likely to have any cards or dashboards for which embedding is enabled at the time they enable embedding for everything?

Reply:
(I'm guessing) it can help us understand if they are re-enabling it after they've used it. Initially we only had the event for the disable event, we added this one later and we used the same schema

Reply:
I don't have significant assumptions to share at this time. I think we'd like to know how they use the toggle.

Do they flip is mostly once during setup? Or also later after they have embedding running? This is where this data could help.